tests/console/sshd.pm

```diff
@@ -50,31 +50,39 @@ sub run {
     assert_script_run("echo $changepwd | chpasswd");
 
     opensusebasetest::select_serial_terminal();
-
-    # Make interactive SSH connection as the new user
-    type_string "ssh -v -l $ssh_testman localhost -t\n";
-    wait_serial('Are you sure you want to continue connecting (yes/no)?', undef, 0, no_regex => 1);
-    type_string "yes\n";
-    wait_serial('Password:', undef, 0, no_regex => 1);
-    type_string "$ssh_testman_passwd\n";
-    wait_serial('sshboy@susetest:~>', undef, 0, no_regex => 1);
-    type_string "export PS1='# '\n";
-
-    # Check that we are really in the SSH session
-    assert_script_run 'echo $SSH_TTY | grep "\/dev\/pts\/"';
-    assert_script_run 'ps ux | egrep ".* \? .* sshd\:"';
-    assert_script_run "whoami | grep $ssh_testman";
-    assert_script_run "mkdir .ssh";
-
-    # Exit properly and check we're root again
-    script_run("exit", 0);
-    assert_script_run "whoami | grep root";
+    if (is_serial_terminal()) {
+        # Make interactive SSH connection as the new user
+        type_string "ssh -v -l $ssh_testman localhost -t\n";
+        wait_serial('Are you sure you want to continue connecting (yes/no)?', undef, 0, no_regex => 1);
```

Comment:
Ahoj @mloviska!

imo it's not about the whitespace, the serial terminal is simply not supported. On [your](https://openqa.suse.de/tests/2306339#step/sshd/18) example the result is empty, but on [supported](https://openqa.suse.de/tests/2308662#step/sshd/26) it contains the whole SSH output. That's why I added the `is_serial_terminal()` if statement.

I was pointed to this test by my manager because of the SSH was recently broken and it was exactly in this part. That's why I'd like to test as it's in real scenario.